index.html

```diff
@@ -10,18 +10,34 @@
     <title>Fred's Personal Portfolio</title>
 </head>
 <body>
+
+    <!--HEADER SECTION -->
+
     <div class="header-container">
         <header class="toolbar-section">
             <nav class="nav-menu">
                 <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
                 <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
+                <ul class="ul-nav-menu">
+                    <li id="portfolio">
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li id="about">
+                        <a href="#">About</a>
+                    </li>
+                    <li id="contact">
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
             </nav>
         </header>
         <section class="headline-section">
             <div class="main-heading">
-                <h1>I’m Mariane Glad to see you!</h1>
+                <h1>I’m Mariane Glad to see you! 
+
+                </h1>
             </div>
-            <div>
+            <div >
                 <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             </div>
```

Comment:
- [ ] Please remove the bar at the top of the header and add a background image to the header.

Your design                       |Figma design               
:--------------------------------: | :-----------------------------------:
![Screenshot from 2022-04-29 20-32-30](https://user-images.githubusercontent.com/33752735/166006495-5d5255e4-72d2-4fe7-ac98-f31eb2887b8a.png)|![Screenshot from 2022-04-29 20-35-00](https://user-images.githubusercontent.com/33752735/166006525-e4756f51-7d2f-42c4-9408-1e78da2b401c.png)